tests/hacker.test.js

```diff
@@ -516,6 +517,124 @@ describe("PATCH update one hacker", function () {
                 });
         });
     });
+
+    // Succeed and change accepted to confirm
+    it("should succeed for hacker to change from accepted to confirmed", function (done) {
+        util.auth.login(agent, storedAccount2, (error) => {
+            if (error) {
+                agent.close();
+                return done(error);
+            }
+            return agent
+                .patch(`/api/hacker/confirmation/${storedHacker2._id}`)
+                .type("application/json")
+                .send({
+                    confirm: true
+                })
+                .end(function (err, res) {
+                    if (err) {
+                        return done(err);
+                    }
+                    res.should.have.status(200);
+                    res.should.be.json;
+                    res.body.should.have.property("message");
+                    res.body.message.should.equal("Changed hacker information");
+                    res.body.should.have.property("data");
+                    chai.assert.equal(JSON.stringify(res.body.data), JSON.stringify({
+                        status: Constants.General.HACKER_STATUS_CONFIRMED
+                    }));
+
+                    done();
+                });
+        });
+    });
+
+    // Succeed and change confirmed to accepted
+    it("should succeed for hacker to change from confirmed to accepted", function (done) {
+        util.auth.login(agent, storedAccount1, (error) => {
+            if (error) {
+                agent.close();
+                return done(error);
+            }
+            return agent
+                .patch(`/api/hacker/confirmation/${storedHacker1._id}`)
+                .type("application/json")
+                .send({
+                    confirm: false
+                })
+                .end(function (err, res) {
+                    if (err) {
+                        return done(err);
+                    }
+                    res.should.have.status(200);
+                    res.should.be.json;
+                    res.body.should.have.property("message");
+                    res.body.message.should.equal("Changed hacker information");
+                    res.body.should.have.property("data");
+                    chai.assert.equal(JSON.stringify(res.body.data), JSON.stringify({
+                        status: Constants.General.HACKER_STATUS_ACCEPTED
+                    }));
+
+                    done();
+                });
+        });
+    });
+
+    // fail for a hacker that's not accepted
+    it("should fail for hacker who is not accepted or confirmed", function (done) {
```

Comment:
Change comment to include "Hacker Confirmation should ..."

Reply:
Do you mean the literal comment, or the message within it(...)?

Reply:
Within `it(...)`, so that if it fails, we understand what failed without even looking at the test function

Reply:
`should fail to update hacker status to confirmed when hacker status is not accepted or confirmed`